src/seedu/addressbook/commands/ViewAllCommand.java

```diff
@@ -37,4 +38,15 @@ public CommandResult execute() {
             return new CommandResult(Messages.MESSAGE_INVALID_PERSON_DISPLAYED_INDEX);
         }
     }
+
+    /**
+     * Returns a concatenated version of the printable strings of each object.
+     */
+    String getPrintableString(Printable... printables) {
+        String detailsString = new String();
```

Comment:
Consider using the StringJoiner class here.

Reference: https://docs.oracle.com/javase/8/docs/api/java/util/StringJoiner.html